apps/docs/src/.vitepress/components/IconLibraryItem.vue

```diff
@@ -17,16 +18,19 @@ const handleCopy = async () => {
   isCopied.value = true;
   setTimeout(() => (isCopied.value = false), 3000);
 };
+
+const tooltipText = computed(() => {
+  if (!isCopied.value) return props.icon.tooltipName;
+  return `${props.icon.tooltipName} - Import copied to clipboard!`;
+});
 </script>
 
 <template>
-  <button type="button" class="icon" @click="handleCopy">
-    <OnyxIcon :icon="props.icon.content" />
-    <span class="icon__tooltip">
-      {{ props.icon.tooltipName }}
-      <div v-if="isCopied">Import copy successful!</div>
-    </span>
-  </button>
+  <OnyxTooltip :text="tooltipText">
```

Comment:
<img width="281" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/214d65e2-45ee-46d5-913a-ba8e3cb72318">
could you use the mode where the tooltip opens on the bottom, like it was originally planned by jannick?